### DISCUSSION TITLE
Add VPN IP address geo location

### DISCUSSION THREAD

**Discussion by GrapeApe230** [qdm12/gluetun#600] [Ideas]
Would it be possible to integrate a service like https://ipgeolocation.io/ to the HTTP control server as an api call? This would be helpful to validate a VPN's config.

**Comment by qdm12** [comment]
Ip information is already fetched from https://ipinfo.io (as logged out).

Please copy pasta your feature request in a github issue and I'll add ~a route `/v1/publicip` with all the content obtained. There is already `/v1/publicip/ip` but I think that one should be removed sometime (when I do a v4 version).~ that information to /v1/publicip/ip
